src/controllers/userController.js

```diff
@@ -100,4 +100,54 @@ const logOut = (req, res) => {
   }
 };
 
-module.exports = { signUp, logIn, logOut };
+/******************************************************
+ * @UPDATEUSER
+ * @route /api/v1/updateuser
+ * @method PUT
+ * @description singUp function for creating new user
+ * @body name, email, password, confirmPassword
```

Comment:
If you put `name, email, password, confirmPassword` in `@body`, does it mean that you have pass them in `req.body`? In that case, do you need to pass `req.user`?